config/jobs/kubernetes/sig-network/sig-network-kind.yaml

```diff
@@ -1,3 +1,60 @@
+presubmits:
+  kubernetes/kubernetes:
+  - name: pull-kubernetes-e2e-kind-conformance-dual
```

Comment:
maybe drop conformance from the name to avoid being misleading, the subset of tests is related to dualstack.